packages/eslint-plugin/docs/rules/no-misused-spread.mdx

```diff
@@ -0,0 +1,141 @@
+---
+description: 'Disallow using the spread operator when it might cause unexpected behavior.'
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-misused-spread** for documentation.
+
+The [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (`...`) is a JavaScript
+feature that can be misused in ways not always reported by TypeScript. This rule disallows using the spread operator on types where
+spreading can lead to unexpected behavior.
+
+This rule disallows the spread operator in the following cases:
```

Comment:
> Do you think we should also explain why each case is disallowed?

[Docs] Yes 🙂. But I'm ok with that being a followup to this PR, IME these kinds of detailed many-cases-explained docs discussions can take a while.

Reply:
Done in #10551